src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,58 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        if (toX > robot.getX()) {
+            switch (robot.getDirection()) {
+                case UP: robot.turnRight();
```

Comment:
please write the operation in the next line after case, not in the same one

Reply:
done